events/2024-11-in-person/index.qmd

```diff
@@ -0,0 +1,142 @@
+---
+title: "In-person, 2-day team brainstorming and planning sessions"
+description: "Details and agendas for our 2-day in-person sessions."
+author: "Luke W. Johnston"
+date: "2024-11-27"
+categories:
+  - in-person
+  - meeting
+  - brainstorming
+  - planning
+---
+
+## Details
+
+-   Informal dinner on Nov 27 and full team dinner on Nov 28.
+-   **Main aims**:
+    -   Understand each other on more personal levels (related to work
+        and Seedcase).
+    -   Not to focus too much on technical aspects, but more on human
+        and social aspects.
+    -   Become informed about other projects that we might collaborate
+        with and how we can contribute.
+
+## Schedule
+
+The schedule is meant as a **guide only**. We might spend more time on
+some topics and less on others.
+
+### Nov 27
+
+| Time  | Item                                                          |
+|-------|---------------------------------------------------------------|
+| 10:00 | Align understandings on Seedcase                              |
+| 12:00 | Lunch                                                         |
+| 13:00 | Personal goals and perspectives                               |
+| 14:00 | ON-LiMiT and DP-Next projects presentations and brainstorming |
+| 15:00 | :coffee: Coffee break                                         |
+| 15:15 | Continue from previous session                                |
+| 16:30 | Wrap Up                                                       |
+| 17:30 | Dinner and social time?                                       |
+
+### Nov 28
+
+| Time  | Item                                   |
+|-------|----------------------------------------|
+| 10:00 | Overview of progress and current state |
+| 11:45 | Lunch                                  |
+| 12:30 | Status update with Annelli             |
+| 14:00 | :coffee: Coffee break                  |
+| 14:15 | Roadmap, deliverables, and milestones  |
+| 16:30 | Wrap Up                                |
+| 17:30 | Team dinner                            |
+
+## Sessions
+
+All brainstorming sessions start with some dedicated individual time to
+think and write down thoughts, then we will discuss them all together.
+
+Maybe use stickies to write down thoughts and then discuss them?
+
+### Align understandings about the current state of Seedcase
```

Comment:
A part of this session could also be @lwjohnst86 presenting the vision for Seedcase as he/you currently see it?

Reply:
I would second that, that would be useful.